sycl/plugins/hip/pi_hip.cpp

```diff
@@ -2202,6 +2202,8 @@ pi_result hip_piQueueFinish(pi_queue command_queue) {
   return result;
 }
 
+pi_result hip_piQueueFlush(pi_queue command_queue) { return PI_SUCCESS; }
```

Comment:
Is there no HIP equivalent that forces the flush? At least, add a comment explaining the implementation